python/ray/experimental/data/read_api.py

```diff
@@ -63,7 +63,7 @@ def from_items(items: List[Any], parallelism: int = 200) -> Dataset[Any]:
     """Create a dataset from a list of local Python objects.
 
     Examples:
-        >>> ds.from_items([1, 2, 3, 4, 5])
+        >>> ray.data.from_items([1, 2, 3, 4, 5])
```

Comment:
When are we planning on moving `data` our of experimental and into the root `ray` package?

Reply:
How about keeping the actual code in experimental until beta? I added an alias so we can refer to it with ray.data so imports can never change.